### ISSUE TITLE
Not reading config file sources

### ISSUE THREAD

**Issue by mnap00** [MohammadYounes/rtlcss#209]
RTLCSS 3.0.0 stopped reading config files.
Version 2.6.2 works as expected.

Also the documentation states that external configs should be named `.rtlcssrc` or `.rtlcssrc.json` but code is looking for `.rtlcssrc` or `.rtlcss.json`.

Line 8 in `lib/config-loader.js`:
```js
var configSources = ['package.json', '.rtlcssrc', '.rtlcss.json']
```

**Comment by MohammadYounes** [comment]
@mnap00 This is a typo in the docs. But you say it has stopped reading the config, I'll check and update you.

**Comment by MohammadYounes** [comment]
@mnap00 Thanks for reporting this. Fixed in v3.1.0

**Comment by mnap00** [comment]
Works for me. Thanks.
